CHANGELOG.rst

```diff
@@ -12,6 +12,14 @@ Change Log
 Unreleased
 ----------
 
+[8.7.0] - 2023-04-14
+--------------------
+
+Added
+~~~~~
+
+* Add ``edx_drf_extensions_version`` to help with rollout of changes in this library across services.
```

Comment:
APM & Services -> Environment -> Modules , `edx_rest_framework_extensions` is listed when installed.  You're right though, it's not clear if the docs even mention that.  I originally learned about it from https://docs.newrelic.com/docs/release-notes/agent-release-notes/python-release-notes/python-agent-80000179/ , but it looks like the Python agent release notes aren't included in their docs search.

Reply:
This is good for spot checking individual services, but not for querying across our an entire system. That is why we left in Django and Python, and have this dashboard: https://onenr.io/0qwLXYxdxw5.

Note: I did open a support ticket to see if it is possible to query that data, and to open a feature request if it is not possible. But, I do not think it is available, and I'm happy to delete this if we learn otherwise.

Reply:
I see, it can tell us that a module is loaded, but not about packages—or their versions.